nvflare/fuel/hci/client/fl_admin_api_spec.py

```diff
@@ -110,21 +112,24 @@ def upload_app(self, app: str) -> FLAdminAPIResponse:
         Currently assumes app is in the upload_dir set in API init.
 
         Args:
-            app: name of the folder in upload_dir to upload
+            app (str): name of the folder in upload_dir to upload
 
         Returns: FLAdminAPIResponse
 
         """
         pass
 
     @abstractmethod
-    def deploy_app(self, app: str, target_type: TargetType, targets: Optional[List[str]] = None) -> FLAdminAPIResponse:
+    def deploy_app(
+        self, run_destination: str, app: str, target_type: TargetType, targets: Optional[List[str]] = None
+    ) -> FLAdminAPIResponse:
```

Comment:
Do we still have deploy_app, start_app?
I thought it should only be "upload_job"

Reply:
I asked Yuhong and he said that we are still going to have those